tensorboard/uploader/server_info_test.py

```diff
@@ -248,6 +248,29 @@ def test_upload_limits_provided_without_max_blob_size(self):
         self.assertEqual(actual, 0)
 
 
+class MaxTensorPointSizeTest(tb_test.TestCase):
+    """Tests for `max_tensor_point_size`."""
+
+    def test_old_server_no_upload_limits(self):
+        info = server_info_pb2.ServerInfoResponse()
+        actual = server_info.max_tensor_point_size(info)
+        self.assertEqual(actual, server_info._DEFAULT_MAX_TENSOR_POINT_SIZE)
+
+    def test_upload_limits_provided_with_max_tensor_point_size(self):
+        info = server_info_pb2.ServerInfoResponse()
+        info.upload_limits.max_tensor_point_size = 42
+        actual = server_info.max_tensor_point_size(info)
+        self.assertEqual(actual, 42)
+
+    def test_upload_limits_provided_without_max_blob_size(self):
+        # This just shows that the proto3 default value of 0 is reported as
+        # usual, not handled as a special case.
+        info = server_info_pb2.ServerInfoResponse()
+        info.upload_limits.max_blob_size = 42
```

Comment:
Maybe add a comment like `# Ensure that upload_limits is set`

Reply:
Done.